rfb/rfb.h

```diff
@@ -763,9 +763,8 @@ extern rfbBool rfbSetNonBlocking(int sock);
 /* websockets.c */
 
 extern rfbBool webSocketsCheck(rfbClientPtr cl);
-extern rfbBool webSocketCheckDisconnect(rfbClientPtr cl);
-extern int webSocketsEncode(rfbClientPtr cl, const char *src, int len, char **dst);
-extern int webSocketsDecode(rfbClientPtr cl, char *dst, int len);
+extern int webSocketsWrite(rfbClientPtr cl, const char *src, int len);
```

Comment:
I can rename the function back to what they were, but the backwards compatibility will still be broken (see libvncserver/sockets.c). I changed the semantics (and signature) of webSocketsEncode (webSocketsWrite) to write directly to the underlying socket instead of hackily replacing the passed buffer with the encoded one. 

Changing back the semantics is not easily done. Also, the return value of the "old" webSocketsEncode being the number of encoded bytes in the buffer, I see no way to make the function robust for large values of "len" (because in this case one does not know how many bytes of the unencoded buffer remain to be sent if not all were processed in the first run).

I will change the signature/name back to what it was before but keep the different semantics.